### PR TITLE
Updated README to include information on the right rust version.

### PR DESCRIPTION
**Pull request by ethanpailes**
Hopefully this will help prevent other people from making the same mistake that I did.

### CODE REVIEW

**Comment by frabrunelle** [comment]
Hi @ethanpailes, we just updated our "Rust Style" document to include a section of which Rust version we currently use: https://github.com/maidsafe/QA/blob/master/Documentation/Rust%20Style.md#rust-version

Could you please update your PR to include that link instead? 🙂

I'm thinking that the first sentence could be updated to something like this:

> To build and use this module locally you need latest stable [Node.js](https://nodejs.org/en/) (& npm) and Rust ([click here to see which version we currently use](https://github.com/maidsafe/QA/blob/master/Documentation/Rust%20Style.md#rust-version)).

or feel free to let me know if you have another idea.

**Comment by ethanpailes** [comment]
@frabrunelle Sure!

File: README.md [hunk 1]
```diff
@@ -10,7 +10,8 @@ safe_app nodejs library.
 
 # Development
 
-To build and use this module locally you need latest stable [NodeJS](https://nodejs.org/en/) (& npm) and [Rust](https://www.rust-lang.org/en-US/) (we recommend [rustup](https://rustup.rs/)).
+To build and use this module locally you need latest stable [NodeJS](https://nodejs.org/en/) (& npm) and [Rust](https://www.rust-lang.org/en-US/) (we recommend [rustup](https://rustup.rs/)). [Click here to see which version we currently use](https://github.com/maidsafe/QA/blob/master/Documentation/Rust%20Style.md#rust-version).
```
Comment (frabrunelle) [inline README.md]: Since it's a separate sentence, I think it would be clearer to say: "Click here to see which version of Rust we currently use"

Comment (ethanpailes) [inline README.md]: I just changed it to explicitly mention rust.

**Comment by ethanpailes** [comment]
Does that work?

**Comment by frabrunelle** [comment]
yes 🙂

**Comment by ethanpailes** [comment]
dope

**Pull request merged**